### PR TITLE
Feat/reservations-rooms-hotel-reservations-preview

### PR DESCRIPTION
**Pull request by Gloomwidow**

### CODE REVIEW

File: server/cmd/reservation-service/service.go [hunk 1]
```diff
@@ -73,17 +102,7 @@ func (s *reservationService) GetClientReservations(ctx context.Context, clientID
 		obj.OfferPreview.OfferTitle = offer.OfferTitle
 		obj.OfferPreview.OfferPreviewPicture = offer.OfferPreviewPicture
 		// todo: add reviews
-		review, err := s.reviewStorage.GetReviewByOwner(ctx, clientID, el.OfferID)
```
Comment (mailo001) [inline server/cmd/reservation-service/service.go:76]: Why you remove this? How you return review id if exist?

File: server/cmd/reservation-service/service.go [hunk 2]
```diff
@@ -12,14 +12,22 @@ type reservationService struct {
 	offerStorage       bookly.OfferStorage
 	reservationStorage bookly.ReservationStorage
 	hotelStorage       bookly.HotelStorage
-	reviewStorage      bookly.ReviewStorage
```
Comment (mailo001) [inline server/cmd/reservation-service/service.go:15]: How do you get informations about review?

File: server/pkg/bookly/reservation.go [hunk 3]
```diff
@@ -24,14 +24,21 @@ type ReservationObject struct {
...
+}
+
 // ReservationInfo holds info about offer for reservation purpose
 type ReservationInfo struct {
 	ReservationID int64     `json:"reservationID"`
 	FromTime      time.Time `json:"from"`
 	ToTime        time.Time `json:"to"`
 	ChildAmount   int       `json:"numberOfChildren"`
 	AdultAmount   int       `json:"numberOfAdults"`
-	ReviewID      *int64    `json:"reviewID,omitempty"`
```
Comment (mailo001) [inline server/pkg/bookly/reservation.go:34]: In database IDs are int64

File: server/cmd/reservation-service/service.go (hunk 1 above)
Comment (Gloomwidow) [reply server/cmd/reservation-service/service.go:76]: I'm not sure how it got removed. I didn't have those changes on local branch. I'll check later.

**Review by mailo001 — APPROVED** [review]

**Pull request merged**